set_up_environment/vs_code.rst

```diff
@@ -0,0 +1,98 @@
+Set up Visual Studio Code
+=========================
+
+.. styled-topic::
+
+  Main questions:
+    How do I set up VS Code to start creating a new course in A+?
+
+  Topics:
+    In this section, we will present the following topics:
+
+    * `VS Code`_
+    * `Installing the recommended VS Code extensions`_
+
+  Material:
+    In this chapter, we do not provide additional material.
+
+  Requirements:
+    You need basic computational skills to install software in the Linux OS.
+
+  Estimated working time:
+    From 30 min to 1 hour.
+
+
+VS Code
+-------
+
+As mentioned in previous chapters of this manual, VS Code is an advanced IDE that provide you with a set of tools 
+designed to improve the experience of creating, editing, and compiling source code. In this particular case, the source
+code is written in RST, and the compiled output are HTML, YAML and static files. Along with the built-in tools
+included in VS Code there are additional extensions designed to facilitate the experience of writing RST. The A+ team 
+developed an extension designed to improve the experience of creating course content. The extension will be discussed in 
+more detail in upcoming sections of this chapter.
+
+The first step to enjoy the benefits of creating course content with the help of VS Code is to install the application 
+in you computer. Installing VS Code is relatively straightforward. You only need to follow the instruction provided in 
+the `download <https://code.visualstudio.com/download>`_ section of the official website.
+
+
+.. _vs-extensions:
+
+Install the recommended VS Code extensions
+------------------------------------------
+
+The `Aplus Tools extension <https://marketplace.visualstudio.com/items?itemName=jaguarfi.aplus-tools>`_ created for VS
+Code does provide a set of functionalities that will reduce the time needed for creating course content, some of the
```

Comment:
does provide -> provides

Reply:
Replace the comma with a period and start a new sentence: Some of the most relevant features are listed below: